### DISCUSSION TITLE
Promedio Controles

### DISCUSSION THREAD

**Discussion by J0SEF4** [IIC2513/Syllabus#40] [General]
Hola!
Tengo una duda de administración, con respecto a como se calcula NC que en el programa se dice:
C1 + C2 + C3 - min(C1,C2,C3) + mean(C1,C2,C3)
Mi duda es que como es este promedio si realizo los 4 controles=
Y no debería haber como un promedio de este calculo por el número de controles que se irán a  considerar?

Saludos y gracias por adelantado!

**Comment by valeeeriquelme** [comment]
Efectivamente la fórmula solo consideraba 3 controles. El cambio sería:
C1 + C2 + C3 + C4 - min(C1,C2,C3, C4) + mean(C1,C2,C3, C4)

**Reply by J0SEF4** [reply]
Disculpa la demora en responder, pero me di cuenta que no hace tanto sentido esa formula en alguna parte hay alguna división? Dado que, si C1,C2,C3,C4 = 4 hipotéticamente. Me quedaría:
(4+4+4+4) -(4) +((4+4+4+4)/4) =16 -4 +4 =16. 
Como es la formula correcta para calcular la nota de los controles entonces?
